pstress/pstress-run.sh

```diff
@@ -90,7 +90,7 @@ echoit(){
 # Kill the server
```

Comment:
can you document the parameter we are passing.

Reply:
Fixed